tngo application structure (#1035)
plotly/dash-docs: dash_docs/chapters/dash_enterprise/application_structure/conda.md

## Conda

Conda is a cross-platform packaging manager that can be used to handle your app's 
Python dependencies. We suggest that Conda should be used if mandated by your organization or if your you apps requires packages that cannot be installed with pip like `cupy` or `cuda`.  

> We recommend using Pip instead of Conda, because it allows for faster app deployments. Conda includes a dependency check used to verify that all installed package requirements are met. This additional check considerably increases the time it takes to deploy an app. This is relevant during development and when deploying larger apps.

Reviewer: ```suggestion
Otherwise, we recommend using Pip instead of Conda because it allows for faster app deployments. Conda's dependency resolver considerably increases the time it takes to install dependencies during development and deployment.
```

{rapids_template}

Initial deployment (conda):

```
git push plotly master  0.09s user 0.08s system 0% cpu 13:51.63 total

```

Subsequent deployment (conda):

```
git push plotly master  0.05s user 0.07s system 0% cpu 16:00.79 total

> ```

Reviewer: This doesn't flow very well and the formatting is confusing. It's not clear what the command or the numbers is referring to. If we'd like to provide some hard numbers about our assertion above, place these numbers in paranthesis in human-readable form:

"... and deployment (30 min vs 15 min)."

Reviewer: I find the relationship between the blockquote and the section below confusing because the block quote makes it feel like the section is ending or we're moving onto a new thought.

OK in both cases, it seems like the conda one takes like 7 minutes longer than the pip one or so? If so, let's keep with the suggestion to move this into the sentence above. Either:

".. and deployment (e.g. 15 min vs 7 min)"

or just "Conda's dependency resolver takes several more minutes to install dependencies during development and deployment."
